config/h5n1-cattle-outbreak.yaml

```diff
@@ -0,0 +1,138 @@
+builds:
+  h5n1-cattle-outbreak: ''
+
+segments:
+  - genome
+  - pb2
+  - pb1
+  - pa
+  - ha
+  - np
+  - na
+  - mp
+  - ns
+
+
+#### Parameters which define the input source ####
+s3_src:
+  name: ncbi
+  metadata: s3://nextstrain-data/files/workflows/avian-flu/h5n1/metadata.tsv.zst
+  sequences: s3://nextstrain-data/files/workflows/avian-flu/h5n1/{segment}/sequences.fasta.zst
+local_ingest: false
+# P.S. To use local ingest files, comment out s3_src and change to local_ingest: joined-ncbi (e.g.)
+
+
+#### Parameters which control large overarching aspects of the build
+# Set a high target_sequences_per_tree to capture all circulating strains, as they will be pruned down
+# as part of the workflow
+target_sequences_per_tree: 10_000
+
+
+#### Config files ####
+reference: config/h5n1/reference_h5n1_{segment}.gb  # use H5N1 references
+genome_reference: config/{subtype}/h5_cattle_genome_root.gb
+auspice_config: config/{subtype}/auspice_config_{subtype}.json
+colors: config/h5n1/colors_h5n1.tsv # use H5N1 colors
+lat_longs: config/h5n1/lat_longs_h5n1.tsv # use H5N1 lat-longs
+include_strains: config/{subtype}/include_strains_{subtype}.txt
+# use cattle-outbreak specific dropped strains for segment + genome trees
+dropped_strains: config/{subtype}/dropped_strains_{subtype}.txt
+clades_file: clade-labeling/h5n1-clades.tsv # use H5N1 clades
+description: config/{subtype}/description_{subtype}.md
+
+
+#### Rule-specific parameters ####
+filter:
+  min_length:
+    FALLBACK:
+      pb2: 2100
+      pb1: 2100
+      pa: 2000
+      ha: 1600
+      np: 1400
+      na: 1270
+      mp: 900
+      ns: 800
+
+  min_date:
+    FALLBACK: 2024
+
+  group_by:
+    FALLBACK: false # no grouping during filter
+
+  exclude_where:
+    FALLBACK: host=laboratoryderived host=ferret host=unknown host=other host=host gisaid_clade=3C.2
+
+
+refine:
+  coalescent: const
+  date_inference: marginal
+
+  clock_filter_iqd:
+    FALLBACK: false
+
+  root:
+    FALLBACK: false
+
+  # For the genome only we use the closest outgroup as the root
+  # P.S. Make sure this strain is force included via augur filter --include
+  # (This isn't needed for the segment builds as we include a large enough time span to root via the clock)
+  genome_root:
+      FALLBACK: A/skunk/NewMexico/24-006483-001/2024
+
+  segment_lengths:
+    FALLBACK:
+      {'pb2': 2341, 'pb1': 2341, 'pa': 2233, 'ha': 1760, 'np': 1565, 'na': 1458, 'mp': 1027, 'ns': 865}
+
+  __clock_std_dev:  &clock_std_dev  0.00211 # YAML anchor so we can reference this value below
+
+  clock_rates:
+    FALLBACK:
+      # The rates for the 8 segments are taken from the GISAID H5N1/2y config
+      pb2: [0.00287, *clock_std_dev]
+      pb1: [0.00264, *clock_std_dev]
+      pa: [0.00248, *clock_std_dev]
+      ha: [0.00455, *clock_std_dev]
+      np: [0.00252, *clock_std_dev]
+      na: [0.00349, *clock_std_dev]
+      mp: [0.00191, *clock_std_dev]
+      ns: [0.00249, *clock_std_dev]
```

Comment:
@jameshadfield BTW, you can do this instead to avoid the bogus extra field:

```yaml
  clock_rates:
    FALLBACK:
      # The rates for the 8 segments are taken from the GISAID H5N1/2y config
      pb2: [0.00287, &clock_std_dev 0.00211]
      pb1: [0.00264, *clock_std_dev]
      pa: [0.00248, *clock_std_dev]
      ha: [0.00455, *clock_std_dev]
      np: [0.00252, *clock_std_dev]
      na: [0.00349, *clock_std_dev]
      mp: [0.00191, *clock_std_dev]
      ns: [0.00249, *clock_std_dev]
```

Avoiding the bogus field is nice because you don't have to worry about it colliding with anything and if a config schema is used (like we do in ncov), it won't cause validation errors.